bundles/org.openhab.binding.folderwatcher/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -123,4 +123,50 @@
 			</parameter>
 		</config-description>
 	</thing-type>
+	<thing-type id="s3bucket">
+		<label>AWS S3 Bucket</label>
+		<description>AWS S3 bucket to be watched</description>
+
+		<channels>
+			<channel id="news3file" typeId="newfile-channel"/>
```

Comment:
Why is the channel labelled differently depending on the thing type? Isn't it always a `newfile` and the type is clear from the thing it originates from?

Reply:
you are right! Thanks, fixed.